src/sagemaker/model_monitor/model_monitoring.py

```diff
@@ -1731,6 +1748,11 @@ def suggest_baseline(
 
         normalized_baseline_output = self._normalize_baseline_output(output_s3_uri=output_s3_uri)
 
+        categorical_drift_method = None
+        if monitoring_config_override and monitoring_config_override.distribution_constraints:
```

Comment:
Is this check required? You are already validating the monitoring config in line 1689

Reply:
Yes Ankit, this check is required to get the value of categorical_drift_method from monitoring_config_override since an empty monitoring_config_override or monitoring_config_override with no distribution_constraints is also being considered as valid.